ivy/functional/ivy/experimental/linear_algebra.py

```diff
@@ -20,6 +21,57 @@ def _check_valid_dimension_size(std):
     ivy.assertions.check_dimensions(std)
 
 
+@to_native_arrays_and_back
+@handle_nestable
+@handle_exceptions
+@handle_array_like_without_promotion
+@handle_array_function
+def eigh_tridiagonal(
+    alpha: Union[ivy.Array, ivy.NativeArray],
+    beta: Union[ivy.Array, ivy.NativeArray],
+    /,
+    *,
+    eigvals_only: bool = True,
+    select: str = 'a',
+    select_range: Optional[Union[Tuple[int, int], List[int], ivy.Array, ivy.NativeArray]] = None,
+    tol: Optional[float] = None,
+) -> Union[ivy.Array, Tuple[ivy.Array, ivy.Array]]:
+    """
+    """
```

Comment:
Could you add docstrings to the function as done with other functions, this is further explained here: https://lets-unify.ai/ivy/deep_dive/docstrings.html